src/api/get-outgoing-filters-soap-api.ts

```diff
@@ -12,9 +13,12 @@ type GetFilterRulesResponse = {
 };
 
 export async function getOutgoingFiltersSoapApi(): Promise<GetFilterRulesResponse> {
-	const response = await soapFetch<unknown, GetFilterRulesResponse>('GetOutgoingFilterRules', {
-		_jsns: 'urn:zimbraMail'
-	}).catch(() => {
+	const response = await legacySoapFetch<unknown, GetFilterRulesResponse>(
+		'GetOutgoingFilterRules',
+		{
+			_jsns: 'urn:zimbraMail'
```

Comment:
We could take the chance to replace all the string literals with the `JSNS` constant provided by `carbonio-ui-soap-lib`